src/components/Modals/Modal/index.tsx

```diff
@@ -1,13 +1,32 @@
 import {FunctionComponent} from "react"
 import "./styles.scss"
 import {ReactComponent as CloseIcon} from "../../../assets/icons/delete.svg"
+import Button from "../../Controls/Button"
+import Divider from "../../UI/Divider"
+import {ReactComponent as WarningIcon} from "../../../assets/icons/warning.svg"
 
 const Modal: FunctionComponent<{
 	show: boolean
 	onClose: () => void
 	zIndex?: number
 	width?: number
-}> = ({show, onClose, children, zIndex, width}) => {
+	title?: string
+	submitButtonText?: string
+	submitButtonDisabled?: boolean
+	onSubmit?: () => void
+	warningMessage?: string
```

Comment:
It's just a pattern spotted from our designs - we do have a repeatable message with a warning icon near to that and this looks pretty much "conventional". I barely believe we would have something like `infoMessage` near to that, could be rather `successMessage`, but that will bring some state handling complexity which we trying to avoid here.

Check https://xd.adobe.com/view/682820cf-2729-46f3-71cc-6cb9a332fc09-1e19/screen/5b234fd6-b206-49fd-823e-c70802aef9d4/
https://xd.adobe.com/view/682820cf-2729-46f3-71cc-6cb9a332fc09-1e19/screen/36f25c1a-5de4-416b-b926-6c2c06247416/
https://xd.adobe.com/view/682820cf-2729-46f3-71cc-6cb9a332fc09-1e19/screen/72d9d438-0bc7-43a0-962d-b8372c0a24c7/
https://xd.adobe.com/view/682820cf-2729-46f3-71cc-6cb9a332fc09-1e19/screen/8b2af675-96e5-44b4-9902-9764d3a90818/

So I would rather suggest keeping it unless something would change in the design pattern. I believe it's quite crucial to keep simple mapping consistency between design and actual components, this will make building new UIs and components much faster